lessonbuilder/tool/src/webapp/css/checklist.css

```diff
@@ -69,29 +69,13 @@ ul#createdChecklistItems .handle, ul#createdChecklistItems .handle:hover {
 .checklistFormInput label{
     margin-left: 0;
     font-weight: bold;
-    color: #555;
+    color: var(--sakai-text-color-2);
     font-size: 110%;
 }
-.checklist-checkbox {
```

Comment:
Removed these styles to use the standard checkbox implementation the rest of the app uses.